node-fs-writefile/cp.js

```diff
@@ -0,0 +1,12 @@
+import fs from 'node:fs/promises';
+
+const filePath = process.argv[2];
+const newFileName = process.argv[3];
+const content = fs.readFile(filePath, 'utf-8');
+const newFile = fs.writeFile(newFileName, await content);
```

Comment:
This works with the `await` here instead of on line 5, but it's unusual. Typically, you would put `await` in front of the function call.